packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -58,13 +58,15 @@ def get_running_agents():
     cluster.post('/mesos/machine/down', json=down_hosts).raise_for_status()
 
     # Wait for replacements
-    test_util.helpers.wait_for_len(get_running_agents, len(cluster.all_slaves), 600)
+    test_util.helpers.wait_for_len(partial(get_running_agents, 'SlaveServerGroup'), len(cluster.slaves), 600)
+    test_util.helpers.wait_for_len(
+        partial(get_running_agents, 'PublicSlaveServerGroup'), len(cluster.public_slaves), 600)
 
     # Reset the cluster to have the replacement agents
-    cluster.slaves = sorted([agent.private_ip for agent in
-                             dcos_launchpad.get_private_agent_ips(state_list=['running'])])
-    cluster.public_slaves = sorted([agent.private_ip for agent in
-                                    dcos_launchpad.get_public_agent_ips(state_list=['running'])])
+    cluster.slaves = sorted([agent.private_ip_address for agent in
+                             get_running_agents('SlaveServerGroup')])
+    cluster.public_slaves = sorted([agent.private_ip_address for agent in
```

Comment:
Why can't we call
DcosCfSimple.get_public_agent_ips()
and
DcosCfSimple.get_private_agent_ips()
https://github.com/dcos/dcos/pull/933/files#diff-3bcac1662148998e548482825464c035R204